src/bin/nydusd/service_controller.rs

```diff
@@ -117,16 +120,33 @@ impl ServiceController {
         };
         let tag = subargs.value_of("fscache-tag");
 
+        let mut threads = 1usize;
+        if let Some(threads_value) = subargs.value_of("fscache-threads") {
+            if let Ok(t) = threads_value.parse::<i32>() {
```

Comment:
Perhaps we can reuse the validator code above.

Reply:
Could you please help to give more hints about the way to reuse code? I haven't found similar pieces of code yet.

Reply:
I think it is like this :)

https://github.com/imeoer/image-service/commit/30fa3f699f30e5e9be8a8f907bcbe0359a7eb145